scalyr_agent/copying_manager/copying_manager.py

```diff
@@ -1338,15 +1339,21 @@ def __consolidate_checkpoints(self, checkpoints):
 
         # clear data folder by removing all worker session checkpoint files.
         checkpoints_glob = os.path.join(
-            self.__config.agent_data_path, "*checkpoints*.json"
+            self.__config.agent_data_path,
+            "{0}*.json".format(WORKER_SESSION_CHECKPOINT_FILENAME_PREFIX),
         )
 
         for path in scalyr_util.match_glob(checkpoints_glob):
-            # do not delete consolidated checkpoint file
```

Comment:
Probably also wouldn't hurt to add additional assert here (e.g. at the very least assert ``assert path.contains(WORKER_SESSION_CHECKPOINT_FILENAME_PREFIX)`` or similar).

Reply:
According to your suggestion there https://github.com/scalyr/scalyr-agent-2/pull/716#discussion_r566150203, the glob has to become safer, so maybe there is no need to add this extra check?